site-scrapers/Harrington/config.js

```diff
@@ -1,18 +1,35 @@
-const site = {
-    name: "Southbridge Community Center",
+const townRestricted = {
+    name: "Harrington Healthcare (town-restricted)",
```

Comment:
This is dumb but could we say something like "Local Residents" instead of "town-restricted"?

Reply:
Done